setup.py

```diff
@@ -3,7 +3,7 @@
 from yangkit.__version__ import __version__
 
 INSTALL_REQUIREMENTS = [
-    'lxml==3.4.4',
+    'lxml==4.9.3',
```

Comment:
i think we should remove the strict version check here

Reply:
Shall we proceed with this - https://github.com/yang-infra/yangkit/pull/12? (Remove this strict version check for all the packages or add '>=' version condition with the older versions?)

Reply:
this is cool when nobody is consuming the yangkit in their frameworks or code.
Just as an example , look at onetopo ..we have to put a condition for 3.6.0 for pyVXR as packages were not compatible.
I have always hesitated  unless you have a full blow UT.

Reply:
ok. i think we should try to put >= instead of just the module name.